backends/bmv2/common/expression.cpp

```diff
@@ -332,6 +332,8 @@ void ExpressionConverter::postorder(const IR::Member* expression)  {
     if (expression->expr->is<IR::Member>()) {
         auto mem = expression->expr->to<IR::Member>();
         auto memtype = typeMap->getType(mem->expr, true);
+        if (memtype->is<IR::Type_Stack>() && mem->member == IR::Type_Stack::next)
+            ::error("%1%: Reading next field, which is always uninitialized", mem);
```

Comment:
Is there a test case for this behavior?

Reply:
Yes, it is part of the PR.

Reply:
It's the file called issue692-bmv2.p4.
Note that this is an error only in this back-end.
It's a warning in the front-end.

Reply:
Got it.